main.cpp

```diff
@@ -20,95 +20,189 @@ You are going to write 10 UDTs in project3.
 */
 
 /*
- example:  
- 
-Thing: Car Wash   
-    5 properties:
-        - number of vacuum cleaners
-        - number of eco-friendly cleaning supplies
-        - stores the amount of water used per week.
-        - stores amount of profit made per week
-        - number of cars serviced per day
-    3 things it can do:
-        - wash and wax car
-        - charge customer
-        - detail the car interior
-
-    Notice that I did not use "has a vacuum cleaner" or "Has eco-friendly cleaning supplies" as one of the properties
-    
-    Writing 'has a ___" checks whether or not your object **has the ability to do something**.
-    Instead, I wrote "number of vacuum cleaners" and "number of eco-friendly cleaning supplies".  
-    
-    These are specific objects or amounts. 
-    
-    In C++ terms, this means to I want you to avoid using 'bool' (has a) as a member variable type.
-    Instead, prefer the other primitive types.
+Thing 1) Coding Language
+5 properties:
+    1) number of data types
+    2) number of function/return types
+    3) number of operators
+    4) number of methods of punctuation
+    5) speed of compiler - bits/sec
+3 things it can do:
+    1) create functions and run functions from libraries
+    2) create data structures to store info
+    3) compile to machine code
  */
 
 /*
-Thing 1)
+Thing 2) Recording Studio
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1) number of channels on console
+    2) number of compressors
+    3) number of time-based effects units
+    4) number of monitoring options
+    5) size of live room - square feet
 3 things it can do:
-    1)
-    2)
-    3)
+    1) Record music
+    2) Mix music
+    3) Host concerts
  */
 
 /*
-Thing 2)
+Thing 3) Electric Guitar
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1) number of strings
+    2) number of frets
+    3) number of pick ups
+    4) number of pick up positions
+    5) number of knobs
 3 things it can do:
-    1)
-    2)
-    3)
+    1) amplify sound from string vibrations
+    2) produce a warm tone
+    3) produce a bright tone 
  */
 
 /*
-Thing 3)
+Thing 4) Refrigerator/Freezer
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1) number of shelves
+    2) number of drawers
+    3) temperature level value - celsius
+    4) size of ice-machine - volume
+    5) size of entire-unit - volume
 3 things it can do:
-    1)
-    2)
-    3)
+    1) chill food
+    2) freeze food
+    3) separate food
  */
 
 /*
-Thing 4)
+Thing 5) struct MidiPads
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1) velocity sensitive
+    2) hardness of rubber
+    3) size of pad
+    4) number of pads
+    5) pads' midi value
 3 things it can do:
-    1)
-    2)
-    3)
+    1) trigger sampler
+    2) send note data
+    3) send velocity data
  */
 
+/*
+Thing 6) struct TransportButtons
+5 properties:
+    1) record button
+    2) play button
+    3) stop button
+    4) fast forward button
+    5) rewind button
```

Comment:
These are objects, not primitives!   You need to pick things that can be represented with `int float double bool char std::string`
- [ ] revise